index.js

```diff
@@ -47,6 +47,13 @@ function Router (opts) {
     this.onBadUrl = null
   }
 
+  if (opts.buildPrettyMeta) {
+    assert(typeof opts.buildPrettyMeta === 'function', 'buildPrettyMeta must be a function')
+    this.buildPrettyMeta = opts.buildPrettyMeta
+  } else {
+    this.buildPrettyMeta = defaultBuildPrettyMeta
```

Comment:
As default, can we skip it instead of defining an empty default?

Reply:
I added a `defaultBuildPrettyMeta` function at the end of `index.js` which does a basic check to ensure a `route` has been passed, then checks for a store object. It lets a user get up and running with it right away without needing to create their own function to parse and return the store.

Reply:
thks for clarification 👍🏽